Source/VolunteerReporting/Events/CaseReportReceived.cs

```diff
@@ -3,20 +3,18 @@
  *  Licensed under the MIT License. See LICENSE in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 using System;
-using Infrastructure.Events;
+using doLittle.Events;
+using Concepts;
 
 namespace Events
 {
     public class CaseReportReceived : IEvent
     {
-        public Guid Id { get; set; }
-        public Guid HealthRiskId { get; set; }
+        public Guid CaseReportId { get; set; }
         public Guid DataCollectorId { get; set; }
```

Comment:
I see that CaseReportFromUnknownDataCollectorReceived has a CaseReportId. The same for InvalidReportFromUnknownDataCollectorReceived. Is the idea that all case reports recieved will en up with a CaseReportId (and a case report in a specific state?) and if .e. data collector is identified, the same casereport gets a new state? Is it the aggregate root that emits events by using appy?

Reply:
The CaseReportId is the actual EventSourceId - typically for the AggregateRoot. A CaseReport could be in different states. For instance, Unknown datacollector vs known. If the datacollector is unknown at one point, later it gets known - the aggregate could reflect this state and apply an event saying the DataCollector has now changed. 

On the read side, we'd move it from the Unknown collection to now be a completely different collection - the known one.

The reason for not using Id as name, was for clarity (which I seem to be inconsistent with two comments down - Origin - more on that two comments down.. :) )